tutorials/optical_tactile_plugin.md

```diff
@@ -171,6 +182,86 @@ is set to `true` and this value is too high, it may slow down the simulation.
 - `<clip>`: These values should be set in a way that the contact surface stays
 in between its far and near values. Otherwise, nothing will be visualized.
 
+## Topics & services
+In order to access the data computed by the plugin and control it during
+simulation, the following topics and services have been advertised:
+
+  `/<namespace>/enable` : Service used to enable and disable the plugin. If disabled,
+  the plugin won't publish messages in the `/normal_forces` nor the `/contacts` topics.
+
+  `/<namespace>/normal_forces` : Topic where a message is published each
+  time the normal forces are computed.
+
+  `/<namespace>/contacts` : Topic where a message is published each time the contacts
+  are computed.
+
+## Performance
+In order to get an idea about the resources consumed by the plugin, some
+experiments have been made by setting the parameters to specific values.
+
+Mainly, there are two parameters that can affect the computational cost of running
+the plugin: the visualization and the contacts resolutions. The following parameters 
+have been tested for a sensor with a size of 20x20x5 mm:
+
+- `<visualization_resolution>`: A value of 20 allows to see an enough
+number of forces and check what the sensor is looking at. However, if we
+set the value to less than 15, lags can be expected when moving around the
+simulation due to the cost of requesting the `/marker` service too many times.
+The simulation itself may will even have problems when loading.
+
+- `<contacts_resolution>`: Like the previous parameter, a value too low can
+cause the simulation to slow down or even not load at all. The following table
```

Comment:
In general, I would refrain from making statements that discourage people from trying out this plugin at all.
```suggestion
cause the simulation to slow down. The following table
```

Reply:
7b40828